bat-utils/lib/runtime-uphold.js

```diff
@@ -0,0 +1,65 @@
+const {
+  default: SDK,
+  RequestClient: Client
+} = require('@uphold/uphold-sdk-javascript')
+const url = require('url')
+const isUUID = require('isuuid')
+
+class AlternativeClient extends Client {
+  constructor (prometheus) {
+    super()
+    this.prometheus = prometheus
+  }
+  path (uri) {
+    const parsed = url.parse(uri)
+    let { path } = parsed
+    const split = path.split('/')
+    let cardinality = 'one'
+    path = split.map((step) => {
+      if (isUUID(step)) {
+        cardinality = 'many'
+        return '{id}'
+      } else {
+        return step
+      }
+    })
+    return {
+      path: path.join('/'),
+      cardinality
+    }
+  }
+  async request (url, method, body, customHeaders = {}) {
+    const { prometheus } = this
+    const {
+      path,
+      cardinality
+    } = this.path(url)
+    const name = 'upholdApi_request_buckets_milliseconds'
```

Comment:
@evq is there a better pattern than this? / is snake case all around preferable? i just went with camel for the first part of the name so to allow for multiword, but kebab would work too, no?

Reply:
yeah I think all snake is more canonical. maybe just `uphold_request_buckets_milliseconds`?

Reply:
ah, right. yeah, i guess i don't need the other ones now because its all going through the uphold api